### PR TITLE
Added Vimeo iframe and NOBUGS banner image

### PR DESCRIPTION
**Pull request by vazovn**

### CODE REVIEW

File: content/news/2024-10-09-ESG-WP5-NOBUGS2024/index.md [hunk 1]
```diff
@@ -8,9 +8,12 @@ subsites: [all-eu, esg]
 main_subsite: eu
 ---
 
+
+<img class="center" src="./NOBUGS_2024.png" width="80%" alt="NOBUGS Welcome page">
```
Comment (bgruening) [inline content/news/2024-10-09-ESG-WP5-NOBUGS2024/index.md:12]: You can add images in markdown and add an alt text if you like.

Comment (vazovn) [reply content/news/2024-10-09-ESG-WP5-NOBUGS2024/index.md:12]: You mean by  ``` Alt text``` . Shall I change it or the html is ok?

Comment (bgruening) [reply content/news/2024-10-09-ESG-WP5-NOBUGS2024/index.md:12]: Yes, please use markdown whenever you can.

**Comment by vazovn** [comment]
Replaced html by markdown

**Pull request merged**